package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "axios": "^0.19.1",
+    "moment": "^2.24.0",
```

Comment:
@StephenAraka we could have a custom function that replaces what this third party library does, I had one but we refactored it with CK and Allan and agreed on how it ought respond.